README.md

```diff
@@ -1,20 +1,34 @@
 # Atlas Basic Terraform Module
 
-This Terraform module creates a basic setup on [MongoDB Atlas](https://www.mongodb.com/products/platform/atlas-database). 
+This Terraform module creates a basic setup on [MongoDB Atlas](https://www.mongodb.com/products/platform/atlas-database).
 
 It creates the following resources:
 
-- A MongoDB Atlas project if the use_existing_project flag is false. If not, you can provide an existing MongoDB Atlas project.
-- One or more database users. If you do not provide any database user, the module creates a default one unless you are using an existing MongoDB Atlas project.
-- Zero or more IP addresses.
-- Zero or more CIDR blocks.
-- A MongoDB Atlas cluster provisioned with AWS, GCP or AZURE provider. It can also be a free tier (TENANT) cluster.
+## Atlas Project
+
+`project_name` by default will try to create a new Atlas project. To reuse an exsting project set `use_existing_project` to `true`
+
+## MongoDB Cluster
+
+A MongoDB Atlas cluster provisioned with AWS, GCP or AZURE provider. It can also be a free tier (TENANT) cluster.
+
+## Database user
+
+Use the `database_users` field to define a list of database user allowed to connect to your MongoDB cluster using [SCRAM](https://www.mongodb.com/docs/manual/core/security-scram/) authentication
+
+If you do not provide any database user, the module creates a default user and password, see [`random_password` resource](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password).
+
+**Note:** when setting `use_existing_project` to `true` no users will be created.
+
+## Access Lists
+
+Use `ip_addresses` or `cidr_blocks` to allow connecting to your MongoDB Cluster from the provided addresses.
 
 You can find detailed information of the module's input and output variables in the [Terraform Public Registry](https://registry.terraform.io/modules/terraform-mongodbatlas-modules/atlas-basic/mongodbatlas/latest).
```

Comment:
we can move this over to the top level description so it doesn't get lost within Access Lists heading

Reply:
done